lib/catalog_of_my_things/genre.rb

```diff
@@ -0,0 +1,15 @@
+class Genre
+  attr_reader :items
+  attr_accessor :id, :name
+
+  def initialize(name, id = Random.rand(1..1_000_000))
+    @name = name
+    @id = id
+    @items = []
+  end
+
+  def add_item(item)
+    @items.push(item)
```

Comment:
We should only push the item if the item is not already in the items array. So the line should be
@items.push(item) unless @items.include?(item)
Please fix this.